packages/pkg-deps.json

```diff
@@ -45,6 +45,9 @@
          "pyserial" : {
             "2" : "pyserial"
          },
+         "pytest": {
+             "3": "python36-pytest"
```

Comment:
@smoser I've tested `run-container` using Debian and CentOS 7 and this was sufficient to get `pytest` running on both. CentOS still fails, but those issues exist in master already.